src/main/java/racingcar/domain/Car.java

```diff
@@ -8,7 +8,7 @@
 
 public class Car {
     private static final int MOVE_MIN_NUM = 4;
-
+    private static final int MIN_POSITION = 0;
     private final String carName;
 
     private int position;
```

Comment:
상수와 인스턴스 변수의 구분을 위해서 공백 라인을 추가해보면 어떨까요?
```suggestion
    private static final int MIN_POSITION = 0;
    
    private final String carName;
    private int position;
```